validator-node-setup.md

```diff
@@ -0,0 +1,157 @@
+# 1. Running a Node
+Here we describe how to run and sync a node using celestia-appd.
+## 1.2 Installing Dependencies
+First, make sure to update and upgrade the OS:
+```sh
+sudo apt update && sudo apt upgrade -y
+```
+Install essential packages:
+```sh
+sudo apt install curl tar wget clang pkg-config libssl-dev jq build-essential git make ncdu -y
+```
+Install GO:
+```sh
+ver="1.17.2"
+cd $HOME
+wget "https://golang.org/dl/go$ver.linux-amd64.tar.gz"
+sudo rm -rf /usr/local/go
+sudo tar -C /usr/local -xzf "go$ver.linux-amd64.tar.gz"
+rm "go$ver.linux-amd64.tar.gz"
+echo "export PATH=$PATH:/usr/local/go/bin:$HOME/go/bin" >> $HOME/.bash_profile
+source $HOME/.bash_profile
+```
+To check if Go was installed correctly run:
+```sh
+go version
+```
+Output should be the version installed:
+```sh
+go version go1.17.2 linux/amd64
+```
+## 1.3 Downloading and Compiling Celestia-App
+The steps below will create a binary file named celestia-appd inside `$HOME/go/bin` folder which will be used later to run the node.
+```sh
+cd $HOME
+rm -rf celestia-app
+git clone https://github.com/celestiaorg/celestia-app.git
+cd celestia-app/
+make install
+```
```

Comment:
We need a one line explanation checking that `make install` is executed successfully 
Example: `celestia-app help `

Reply:
Added an explanation on how to check if the binary was successfully compiled using --help.